docs/2.0/reporting/gradle-autoconfigure-junit5.adoc

```diff
@@ -0,0 +1,70 @@
+For Gradle users, the `allure-gradle` plugin is available. The plugin autoconfigures all of the required dependencies:
+
+[source, groovy, linenums]
+.build.gradle
+----
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.junit.platform:junit-platform-gradle-plugin:1.0.2'
+        classpath 'io.qameta.allure:allure-gradle:2.4'
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'base'
+apply plugin: 'org.junit.platform.gradle.plugin'
+apply plugin: 'io.qameta.allure'
+
+sourceCompatibility = 1.8
+
+repositories {
+    jcenter()
+    mavenCentral()
+}
+
+dependencies {
+    compile 'io.qameta.allure:allure-gradle:2.4'
+    testCompile(
+            'org.junit.jupiter:junit-jupiter-api:5.0.1',
+            'io.qameta.allure:allure-gradle:2.4'
+    )
+    testRuntime(
+            'org.junit.jupiter:junit-jupiter-engine:5.0.1',
+            'org.junit.vintage:junit-vintage-engine:4.12.1',
+            'org.junit.platform:junit-platform-launcher:1.0.2',
```

Comment:
only `junit-jupiter-engine` is required during test runtime

Reply:
removed all the rest